features/facebookImport/src/importer/data-importers/name-importer.js

```diff
@@ -1,11 +1,14 @@
-import DirectKeyDataImporter from "./direct-key-data-importer.js";
+import { readJSONDataObject } from "../importer-util.js";
 
-export default class NameImporter extends DirectKeyDataImporter {
-    constructor() {
-        super(
-            "profile_information/profile_information.json",
-            "profile_v2",
-            "name"
-        );
+const fileName = "profile_information/profile_information.json";
+
+export default class NameImporter {
+    async _readLanguageData(id, zipFile) {
+        return readJSONDataObject(fileName, "profile_v2", zipFile, id);
+    }
+
+    async import({ id, zipFile }, facebookAccount) {
+        const profileData = await this._readLanguageData(id, zipFile);
+        facebookAccount.name = profileData?.name?.full_name;
```

Comment:
It's OK to guard ourselves from missing data, but if the profile name is missing there's something seriourly wrong with the file, and it would probably need to fail at a whole another level...

Reply:
I removed the check to let the importer fail. That way at least we get an entry in the report.